### ISSUE TITLE
Keyboard Binding Using Super Key

### ISSUE THREAD

**Issue by Russell-S** [linuxmint/Cinnamon#2766]
I've just installed Linux Mint 16 x64 Cinnamon Edition and there appears to be a problem assigning the Windows/Super key in 'Keyboard Shortcuts'.

The default behaviour is for the left Super key to activate the mint menu but what I am trying to do is make the left Super key activate the Cinnamon Scale mode.
Therefore I have deleted the binding for the menu button to free up the left Super key. However, now it will not allow me to re-assign the Super key on it's own to anything. I can assign the Super key + another key without issue but I cannot assign either of the Super keys just on their own. I cannot even get back to the original setup where the left Super key opened the mint menu.

I know this worked in Mint 15 Cinnamon (and previous versions) so I am assuming this may be a bug in Mint 16

Basically, when you click on the 'unassigned' Keyboard binding it changes to 'New accelerator' but then nothing happens when you press either Super key on it's own.
